src/servers/FtpServer.cc

```diff
@@ -700,10 +700,11 @@ Ftp::Server::parseOneRequest()
     Must(parsed && cmd.length());
     consumeInput(tok.parsedSize()); // TODO: Would delaying optimize copying?
 
-    debugs(33, 2, ">>ftp " << cmd << (params.isEmpty() ? "" : " ") << params);
-
     cmd.toUpper(); // this should speed up and simplify future comparisons
 
+    traceProtocol(9, "FTP Client REQUEST", clientConnection,
```

Comment:
Please report the received `cmd` before we change it. These message dumps should use as much _raw/actual_ data as practical to improve triage experience.